src/formBuilder/defaults/shortAnswerInputs.js

```diff
@@ -176,6 +176,33 @@ function CardShortAnswerParameterInputs({
         }}
         className='card-modal-select'
       />
+      <h4>
+        Column Size{' '}
+        <a
+          href='https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout'
+          target='_blank'
+          rel='noopener noreferrer'
+        >
+          <Tooltip
+            id={`${elementId}_column_size`}
+            type='help'
+            text='Set the column size of the input'
+          />
+        </a>
+      </h4>
+      <Input
+        value={parameters['ui:column'] ? parameters['ui:column'] : ''}
+        placeholder='Column size'
+        key='ui:column'
+        type='number'
+        onChange={(ev: SyntheticInputEvent<HTMLInputElement>) => {
+          onChange({
+            ...parameters,
+            'ui:column': ev.target.value,
```

Comment:
I can't remember if setting the input component's type to "number" actually means that `ev.target.value` is actually a number. If it is not, should we parse this to a number?

Reply:
It seems the input only accepts numbers as is. So `e.target.value` will be a number.

Reply:
Great!